cmake/FDBComponents.cmake

```diff
@@ -162,6 +162,8 @@ endif()
 ################################################################################
 
 set(SSD_ROCKSDB_EXPERIMENTAL ON CACHE BOOL "Build with experimental RocksDB support")
+set(PORTABLE_ROCKSDB ON CACHE BOOL "Build RocksDB with `-march=native`") # Set this to OFF if you want to compile RocksDB with
```

Comment:
`OFF` means `-march=native`, right?

BTW, can you fix the comment?

Reply:
That is correct, I've updated the comment to be accurate.